src/pytest_mock_resources/fixture/database/redis.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from pytest_mock_resources.compat import redis
+from pytest_mock_resources.container import get_docker_host
+from pytest_mock_resources.fixture.database.generic import assign_fixture_credentials
+
+
+def create_redis_fixture(**kwargs):
```

Comment:
can you add a docstring to this which documents the scope kwarg, and then make sure it shows up in the api.rst doc section?